src/group_theory/subgroup.lean

```diff
@@ -353,6 +353,17 @@ begin
   rw [← subgroup.coe_mk H y hy, subsingleton.elim (⟨y, hy⟩ : H) 1, subgroup.coe_one],
 end
 
+@[to_additive] instance fintype_bot : fintype (⊥ : subgroup G) := ⟨{1},
+by {rintro ⟨x, ⟨hx⟩⟩, exact finset.mem_singleton_self _}⟩
+
+@[simp] lemma card_trivial_add : fintype.card (⊥ : add_subgroup A) = 1 :=
+fintype.card_eq_one_iff.2
+  ⟨⟨(0 : A), set.mem_singleton 0⟩, λ ⟨y, hy⟩, subtype.eq $ add_subgroup.mem_bot.1 hy⟩
```

Comment:
Can you add a comment stating that `@[to_additive]` doesn't work here?

Also, these lemmas should probably be called `[add_]subgroup.card_trivial`.

Reply:
Done.